### PR TITLE
Refactoring components from App.svelte

### PR DESCRIPTION
**Pull request by dkumiszhan**
This PR refactors App component into smaller independent components.

### CODE REVIEW

File: src/components/Config.svelte [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    //         messages = messages.slice(messages.length - $maxNumberOfMessages, messages.length);
+    //     }
+
+    //     if ($env == ENV_PROD) {
+    //         if (conn != null) {
+    //             conn.close();
+    //         }
+    //         messages = [];
+    //     }
+    // }
```
Comment (glaslos) [inline src/components/Config.svelte:27]: Remove code that you don't need anymore

File: src/components/Modal.svelte [hunk 2]
```diff
@@ -25,28 +24,25 @@
...
-        env = currentEnv;
-        dialog.close();
-    }
+    // function applyConfig() {
+    //     // Let parent know about config update
+    //     dispatch('configChange');
+    //     maxNumberOfMessages.set(currentNumberOfMessages);
+    //     env = currentEnv;
+    //     dialog.close();
+    // }
```
Comment (glaslos) [inline src/components/Modal.svelte:33]: Remove unused code

**Review by glaslos — CHANGES_REQUESTED** [review]

File: src/components/Config.svelte (hunk 1 above)
Comment (dkumiszhan) [reply src/components/Config.svelte]: done

File: src/components/Modal.svelte (hunk 2 above)
Comment (dkumiszhan) [reply src/components/Modal.svelte]: done

**Comment by glaslos** [comment]
Great work 💪 once you resolve the merge conflicts we can merge the PR

**Comment by dkumiszhan** [comment]
> Great work 💪 once you resolve the merge conflicts we can merge the PR

Done

**Review by glaslos — APPROVED** [review]

**Pull request merged**